commands/applycmd.go

```diff
@@ -24,7 +24,12 @@ func GetApplyRunner(provider provider.Provider, loader manifestreader.ManifestLo
 	w := &ApplyRunnerWrapper{
 		applyRunner: applyRunner,
 		factory:     provider.Factory(),
+		ioStreams:   ioStreams,
+		applyCRD:    false,
 	}
+	cmd := applyRunner.Command
+	cmd.Flags().BoolVar(&w.applyCRD, "apply-inventory-crd", false,
```

Comment:
I think something like `install-resource-group` would be a better name for the flag here as it mirrors the name of the command we already have.

Reply:
Done